SwiftPackage/Sources/AssessmentModelUI/Resources/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,68 @@
+/* let format = NSLocalizedString("(Maximum %1$d characters)", bundle: .module, comment: "Maximum characters for test input") */
+"(Maximum %1$d characters)" = "(Maximum %1$d characters)";
+
+/* Text("Begin in...", bundle: .module) */
+"Begin in..." = "Begin in...";
+
+/* Text("Continue later", bundle: .module) */
+"Continue later" = "Continue later";
+
+/* let doneTitle = NSLocalizedString("Done", bundle: .module, comment: "Done button for entering text") */
+"Done" = "Done";
+
+/* Text("Exit", bundle: .module) */
+"Exit" = "Exit";
+
+/* Text("Hello, World") */
+"Hello, World" = "Hello, World";
+
+/* return NSLocalizedString("Hours", bundle: .module, comment: "Field label for number of hours in a duration question") */
+"Hours" = "Hours";
+
+/* Text("Lorem ipsum dolor sit amet, consectetur adipiscing elit.") */
+"Lorem ipsum dolor sit amet, consectetur adipiscing elit." = "Lorem ipsum dolor sit amet, consectetur adipiscing elit.";
```

Comment:
Is this file a sample/placeholder/test file? I assume we don't intend to localize lorem ipsum text (and I presume we're not actually showing it to participants anyway)....

Reply:
Auto-generated